src/models/index.ts

```diff
@@ -38,22 +39,31 @@ if (config.use_env_variable) {
   sequelize = new Sequelize(config.database, config.username, config.password, config);
 }
 
+const db = {
+  sequelize,
+  Sequelize,
+  CaseAudit: CaseAuditFactory(sequelize),
+};
+
+// ToDo: this code is not working in typescript. We need to figure out a better way to load the models dynamically
 fs.readdirSync(__dirname)
   .filter(file => {
     return file.indexOf('.') !== 0 && file !== basename && file.slice(-3) === '.js';
   })
   .forEach(file => {
-    const model = sequelize.import(path.join(__dirname, file));
-    db[model.name] = model;
+    if (file !== 'case-audit.js') {
+      const model = sequelize.import(path.join(__dirname, file));
+      db[model.name] = model;
+    }
```

Comment:
Yeah, I thought this file specific would be hard to translate. Sequelize has an issue with circular dependencies in associations, and this is the way they suggested (on a GitHub discussion) to make it work. Feels hacky.